symfony/dependency-injection/CVE-2019-10910.yaml

```diff
@@ -0,0 +1,35 @@
+title:     "CVE-2019-10910: Check service IDs are valid"
+link:      https://symfony.com/cve-2019-10910
+cve:       CVE-2019-10910
+branches:
+    2.7.x:
+        time:     2019-04-16 12:48:27
+        versions: ['>=2.7.0', '<2.7.51']
```

Comment:
The `>=2.7.0` should be removed here, so that the range covers all older versions too, which are also affected

Reply:
No, the range is only for the 2.7 branch, so removing the lower constraint is not a good idea.

Reply:
but then, we need to have ranges for all older affected versions too